ValveResourceFormat/IO/ModelExtract.ValveModel.cs

```diff
@@ -684,12 +684,17 @@ Lazy<KVObject> MakeLazyList(string className)
 
         if (physAggregateData is not null)
         {
-            foreach (var physicsPart in physAggregateData.Parts)
+            var boneNames = this.physAggregateData.Data.GetArray<string>("m_boneNames");
```

Comment:
The list could be empty. You might get away with adding an empty string as the `parent_bone` property if that's the case.